flutter/example/pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
     sdk: flutter
   sentry_flutter: any
   universal_platform: ^1.0.0-nullsafety
+  feedback: 2.0.0
```

Comment:
I saw it coming :D

Reply:
I'll remove it later. It's nice for testing attachments though.

Reply:
we dont have to, its an example :)